docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -1329,6 +1329,24 @@ SELECT *
 FROM example.testdb.customer_orders FOR TIMESTAMP AS OF TIMESTAMP '2022-03-23 09:59:29.803 Europe/Vienna'
 ```
 
+You can use a date to specify a point a time in the past for using a snapshot of a table in a query.
+Assuming that the session time zone is `Europe/Vienna` the following queries are equivalent:
+
+```
+SELECT *
+FROM example.testdb.customer_orders FOR TIMESTAMP AS OF DATE '2022-03-23'
+```
+
+```
+SELECT *
+FROM example.testdb.customer_orders FOR TIMESTAMP AS OF TIMESTAMP '2022-03-23 00:00:00'
+```
+
+```
+SELECT *
+FROM example.testdb.customer_orders FOR TIMESTAMP AS OF TIMESTAMP '2022-03-23 00:00:00.000 Europe/Vienna'
+```
```

Comment:
We need to scroll for reading the entire query. I think it would be better to wrap after the table name. Follow-up is fine because this format is same as the existing query. 

<img width="785" alt="Screenshot 2023-09-06 at 17 39 00" src="https://github.com/trinodb/trino/assets/6237050/caafd054-45b1-412f-9b66-71dada83a559">